src/components/MovieList/MovieListComponent.js

```diff
@@ -14,40 +14,42 @@ const MovieListComponent = (props) => {
     });
   };
 
-  const deleteMovie =  movie => {
+  const deleteMovie = movie => {
     store.dispatch(deleteMovieAsync(movie)).then(() => {
       props.fetchMovies();
     });
   };
 
   return (
-    <>
-      <div className="movies-count">{movies.length} movies found</div>
-      <div className="movie-list">
-        {movies.map(value =>
-           (
-            <MovieCardComponent
-              setMovieDetails={props.setMovieDetails}
-              movie={value}
-              key={value.id}
-              editMovie={editMovie}
-              deleteMovie={deleteMovie}
-            />
-          )
-        )}
-      </div>
-    </>
+    <div>
+      {!movies.length && (
+        <div className="info-message">
+          <h1>No Movie Found</h1>
+        </div>
+      )}
+
+      {movies.length && (
```

Comment:
let's change your condition and check more then 0 values)
because movies length might be 0

`{movies.length > 0 && (.........)}`